tests/Test/Rfc3339.elm

```diff
@@ -275,10 +256,10 @@ printing =
                     |> Expect.equal (Ok "00:00:00")
         , test "local time right after midnight" <|
             \() ->
-                "00:00:00.00001"
+                "00:00:00.001"
                     |> Rfc3339.parse
                     |> Result.map Rfc3339.toString
-                    |> Expect.equal (Ok "00:00:00.00001")
+                    |> Expect.equal (Ok "00:00:00.001")
```

Comment:
Thinking we should add a test that acknowledges that we only support milliseconds

Reply:
As in - input with more precision, test that it comes out with less?

Reply:
Exactly!